src/chartsFactory.jsx

```diff
@@ -61,7 +61,13 @@ module.exports = function (chartType, Highcharts){
     },
 
     render: function (){
-      let props = this.props;
+      let {
+          callback: callbackIgnored,
+          config: configIgnored,
+          isPureConfig: isPureConfigIgnored,
+          neverReflow: neverReflowIgnored,
```

Comment:
Sure. I was trying to be explicit (practice we implemented at Rapid7 to identify variable assignment that was specifically unused), but no problem. I'll have it updated in a jiffy.